soccer/gameplay/GameplayModule.hpp

```diff
@@ -53,7 +53,12 @@ namespace Gameplay
 			/**
 			 * @brief Loads a playbook file to enable specified plays
 			 */
-			void loadPlaybook(const std::string &playbookFile);
+			void loadPlaybook(const std::string &playbookFile, bool isAbsolute = false);
+
+			/**
+			 * @brief Saves the currently enabled plays to a playbook file
+			 */
+			void savePlaybook(const std::string &playbookFile, bool isAbsolute = false);
```

Comment:
Go ahead and make a note here and for the above function about what `isAbsolute` does.